src/equations/acoustic_perturbation_2d.jl

```diff
@@ -17,59 +49,65 @@ Volume 188, Issue 2,
 2003,
 [DOI: 10.1016/S0021-9991(03)00168-2](https://doi.org/10.1016/S0021-9991(03)00168-2)
 """
-struct AcousticPerturbationEquations2D{RealT<:Real} <: AbstractAcousticPerturbationEquations{2, 3}
-  v_mean::SVector{2, RealT}
-  c_mean::RealT
-  rho_mean::RealT
+struct AcousticPerturbationEquations2D{RealT<:Real} <: AbstractAcousticPerturbationEquations{2, 7}
+  v_mean_global::SVector{2, RealT}
+  c_mean_global::RealT
+  rho_mean_global::RealT
 end
 
-function AcousticPerturbationEquations2D(v_mean::NTuple{2,<:Real}, c_mean::Real, rho_mean::Real)
-  return AcousticPerturbationEquations2D(SVector(v_mean), c_mean, rho_mean)
+function AcousticPerturbationEquations2D(v_mean_global::NTuple{2,<:Real}, c_mean_global::Real,
+                                         rho_mean_global::Real)
+  return AcousticPerturbationEquations2D(SVector(v_mean_global), c_mean_global, rho_mean_global)
 end
 
-function AcousticPerturbationEquations2D(; v_mean::NTuple{2,<:Real}, c_mean::Real, rho_mean::Real)
-  return AcousticPerturbationEquations2D(SVector(v_mean), c_mean, rho_mean)
+function AcousticPerturbationEquations2D(; v_mean_global::NTuple{2,<:Real}, c_mean_global::Real,
+                                         rho_mean_global::Real)
+  return AcousticPerturbationEquations2D(SVector(v_mean_global), c_mean_global, rho_mean_global)
 end
 
 
 get_name(::AcousticPerturbationEquations2D) = "AcousticPerturbationEquations2D"
-varnames(::typeof(cons2cons), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_prime")
-varnames(::typeof(cons2prim), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_prime")
+varnames(::typeof(cons2cons), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_prime",
+                                                                    "v1_mean", "v2_mean", "c_mean", "rho_mean")
+varnames(::typeof(cons2prim), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_prime",
+                                                                    "v1_mean", "v2_mean", "c_mean", "rho_mean")
 
 
-"""
-    initial_condition_constant(x, t, equations::AcousticPerturbationEquations2D)
+# Convenience functions for retrieving state variables, mean variables and global mean values
+function get_state_vars(u, equations::AcousticPerturbationEquations2D)
+  return u[1], u[2], u[3]
+end
 
-A constant initial condition.
-"""
-function initial_condition_constant(x, t, equations::AcousticPerturbationEquations2D)
-  v1_prime = 0.0
-  v2_prime = 0.0
-  p_prime = 0.0
+function get_mean_vars(u, equations::AcousticPerturbationEquations2D)
+  return u[4], u[5], u[6], u[7]
+end
 
-  return SVector(v1_prime, v2_prime, p_prime)
+function global_mean_values(equations::AcousticPerturbationEquations2D)
```

Comment:
```suggestion
function global_mean_vars(equations::AcousticPerturbationEquations2D)
```
To be (slightly) more consistent with `get_state_vars` etc.?

Also, I think it would be good to have a short docstring for this function.

Reply:
Done in 4643b42.